Submitted By:            DJ Lucas (dj AT linuxfromscratch DOT org)
Date:                    2005-10-06
Initial Package Version: 1.5.0
Origin:                  JDK-6.0 (Mustang) snapshot
                         Rediffed for 1.5.0.11 by Randy McMurchy
Upstream Status:         Fixed in 6.0
Description:             Fixes syntax for use with nptl.


diff -Naur jdk-1.5.0.11-orig/deploy/src/javaws/solaris/native/system_md.c jdk-1.5.0.11/deploy/src/javaws/solaris/native/system_md.c
--- jdk-1.5.0.11-orig/deploy/src/javaws/solaris/native/system_md.c	2007-03-19 05:06:23.000000000 +0000
+++ jdk-1.5.0.11/deploy/src/javaws/solaris/native/system_md.c	2007-03-19 05:08:55.000000000 +0000
@@ -880,3 +880,8 @@
 
   return strcat(temp, dbg_suffix);
 }
+
+char *sysGetNetProperty() {
+  return NULL;
+}
+
